mathics/builtin/list/rearrange.py

```diff
@@ -692,7 +693,7 @@ class Riffle(Builtin):
       <dd>inserts a copy of $x$ between each element of $list$.
 
       <dt>'Riffle[{$a1$, $a2$, ...}, {$b1$, $b2$, ...}]'
-      <dd>interleaves the elements of both lists, returning '{$a1$, $b1$, $a2$, $b2$, ...}'.
+      <dd>interelements the elements of both lists, returning '{$a1$, $b1$, $a2$, $b2$, ...}'.
```

Comment:
interelments -> inter-elements

Reply:
Interleaves?

Reply:
Oops - you are absolutely correct! This is an English word so it should stay as is.

Reply:
Yep, I changed it accidentally.